### DISCUSSION TITLE
Which files do I move for 1.20.1 Forge in Eclipse?

### DISCUSSION THREAD

**Discussion by Fruitcakei** [Luke100000/minecraft-comes-alive#727] [Q&A]
Hi! I'm currently trying to create a child mod, a small add-on to this one that would create skill levels with improving AI, using Eclipse. I have a few questions:
1) My main question: which files do I move over?
2) Do I put the whole MCA file directly under src/main/java, do some of them go under src/main/resources, or do they go under various folders?
3) Can I do this in Eclipse, or do I just have to give up on it and start over in GitHub?
4) Does this still need JDK 17 to run, or is it compatible with JDK 20 now?

If you have answers to any of these, please do leave a comment! Thank you!

**Comment by Luke100000** [comment]
I do not know how addons would work best. Here is a repo who did it https://github.com/martinkoeth/MCA-Quests/blob/main/build.gradle tho I do not know its current state. On our discord is a user who tried it, maybe ask there too.

If I have some time I could also take a look at how addons would work best.

If you just want to play around and make your own version of mca, clone the whole repo. Eclipse should recognize gradle.

Java is mostly backwards compatible, it should run fine, but since Minecraft is shipped with 18 it makes sense to use that one.

Of course you can use eclipse, GitHub is not an editor at all.
